packages/gdl-frontend/pages/books/translations.js

```diff
@@ -107,20 +107,33 @@ class TranslationCard extends React.Component<
           >
             <Trans>Sync</Trans>
           </LoadingButton>
-          <Button
-            color="primary"
-            href={translation.crowdinUrl}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <Trans>Edit</Trans>
-          </Button>
+          <TranslateButton
+            id={translation.id}
+            lang={translation.translatedFrom.code}
+          />
```

Comment:
Legge til en knapp her som linker til crowdin for å kunne oversette der (for desktop mode). Så egentlig gjeninnføre den knappen som lenket brukeren til crowdin og gi den et fornuftig navn, feks: "Editer i crowdin" elns.